ggml/src/lib.rs

```diff
@@ -147,6 +147,12 @@ impl Context {
         self.new_tensor_raw(raw)
     }
 
+    /// Unknown, aside from the obvious. It's transposing something!
```

Comment:
love that documentation from upstream

Reply:
It's not from upstream, it's from me. :)

The original comment was `// ggml_transpose`

Reply:
Yeah that's what I mean - upstream's documentation is... lacking 😅

Reply:
Oh, derp. My brain is slightly fried at the moment.